public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# Allow all crawlers
+User-agent: *
+Allow: /
```

Comment:
I am not sure if we want to block any AI from crawling the app, but on my own site, I use this, `User-agent: GPTBot
Disallow: /`. There's probably others.

```suggestion
User-agent: GPTBot
Disallow: /

# Allow all crawlers
User-agent: *
Allow: /
```